src/main/java/com/libertymutualgroup/herman/aws/ecs/broker/rds/RdsBroker.java

```diff
@@ -48,10 +48,11 @@
 
 public class RdsBroker {
 
-    public static final String AURORA_ENGINE = "aurora";
-    public static final String POSTGRES_ENGINE = "postgres";
-    public static final String MYSQL_ENGINE = "mysql";
-    public static final String CIPHER_PREFIX = "{cipher}";
+    private static final String AURORA_ENGINE = "aurora";
+    private static final String POSTGRES_ENGINE = "postgres";
+    private static final String MYSQL_ENGINE = "mysql";
+    private static final String SQL_SERVER_ENGINE = "sqlserver-se";
```

Comment:
Hm, not used anywhere? This should probably be a list if it is:
sqlserver-ee
sqlserver-se
sqlserver-ex
sqlserver-web

Reply:
not used anywhere so can remove. important bit is the SqlServerClient gets used if the engine contains "sqlserver"